packages/cysync-core/src/pages/OnBoarding/Wallet/WalletActions.tsx

```diff
@@ -0,0 +1,340 @@
+      align="center"
+      gap={40}
+      direction="column"
+      height="full"
+    >
+      <Flex align="center" direction="column">
+        <Typography
+          width={{
+            def: 582,
+            lg: 1141,
+          }}
+          $textAlign="center"
+          fontSize={{
+            def: 20,
+            lg: 24,
+          }}
+          variant="h4"
+          color="heading"
+          mb={1}
+        >
+          <LangDisplay text={title} />
+        </Typography>
+        <Typography
+          display={{
+            def: 'none',
+            lg: 'block',
+          }}
+          width={964}
+          $textAlign="center"
+          variant="h5"
+          color="muted"
+          mb={1}
+        >
+          <LangDisplay text={subTitle} />
+        </Typography>
+      </Flex>
+      <Flex gap={20} px={{ def: '20', lg: '150' }}>
+        <DialogBox width="full">
+          <DialogBoxBody height="full">
+            <Image width={45} src={addWalletIcon} alt="importWalletIcon" />
+            <Flex gap={48} direction="column" height="full">
+              <Typography
+                $textAlign="center"
+                variant="h5"
+                color="heading"
+                mb={1}
+              >
+                <LangDisplay text={createWallet.title} />
+              </Typography>
+              <Container
+                display={{
+                  def: 'none',
+                  lg: 'flex',
+                }}
+                justify="flex-start"
+                height="full"
+                $bgColor="list"
+                direction="column"
+              >
+                <ListContainer p={4} direction="column" width="full">
+                  {createWallet.list.map((item, index) => (
+                    <ListItem
+                      key={`create-wallet-list-${index + 1}`}
+                      width="full"
+                    >
+                      <Typography
+                        variant="h6"
+                        color="muted"
+                        $textAlign="left"
+                        mb={2}
+                      >
+                        {item}(
+                        <span
+                          style={{
+                            background: theme.palette.golden,
+                            WebkitTextFillColor: 'transparent',
+                            WebkitBackgroundClip: 'text',
+                          }}
+                        >
+                          ?
+                        </span>
+                        )
+                      </Typography>
+                    </ListItem>
+                  ))}
+                </ListContainer>
+              </Container>
+            </Flex>
+          </DialogBoxBody>
+          <DialogBoxFooter>
+            <Link to="/create-new-wallet">
+              <Button variant="primary">
+                <LangDisplay text={createWallet.button} />
+              </Button>
+            </Link>
```

Comment:
Don't use hardcoded links, use `routes` object. Also `createNewWallet` should be a popup and not a page.